### ISSUE TITLE
ExplicitTimeIntegration test takes too long

### ISSUE THREAD

**Issue by vhirtham** [nutofem/nuto#75]
If I compile in debug mode and run the tests I have to wait 10minutes (636s) just for the explicit time integration test. Thats 58% of the whole testing time (1080s). Is there any way to reduce the execution time?

**Comment by pmueller2** [comment]
Yes. Since the test involves a simple wave equation and the solution is known it is probably enough to do only some time steps and then compare the result. In my opinion it is also not necessary to use such a fast varying load (step function type) for testing puposes.
This may be done along with the changes suggested in #45 (and maybe also be done by me).

**Comment by Psirus** [comment]
Once this is done, I suggest moving it to integrationtests, since it is actually a test and should influence our coverage.
